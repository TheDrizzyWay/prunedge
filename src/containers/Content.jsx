import React from 'react';
import Dashboard from '../components/Dashboard';
import DocumentsView from '../components/DocumentsView';
import RecentView from '../components/RecentView';

const Content = () => {
    return (
        <div className="content">
            <div className="content__wrapper">
                <Dashboard />
                <DocumentsView />
                <RecentView />
            </div>
        </div>
    );
}

export default Content;
