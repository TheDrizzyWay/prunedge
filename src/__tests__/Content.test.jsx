import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Content from '../containers/Content';

configure({ adapter: new Adapter() });

describe('Content component', () => {
  it('should render Content', () => {
    const wrapper = shallow(
      <Content />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
