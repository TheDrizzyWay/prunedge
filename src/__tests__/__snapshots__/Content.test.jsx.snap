// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content component should render Content 1`] = `ShallowWrapper {}`;

exports[`Menu component should render Menu 1`] = `ShallowWrapper {}`;
